Formula/mariadb.rb

```diff
@@ -33,8 +33,8 @@ class Mariadb < Formula
     # Need patch to remove MYSQL_SOURCE_DIR from include path because it contains
     # file called VERSION.
     # https://github.com/Homebrew/homebrew-core/pull/76887#issuecomment-840851149
-    # Reported upstream at https://jira.mariadb.org/browse/MDEV-7209 - this fix can be
-    # removed once that issue is closed and the fix has been merged into a stable release.
+    # Originally reported upstream at https://jira.mariadb.org/browse/MDEV-7209,
+    # but only partially fixed.
     patch :DATA
```

Comment:
Can we use the upstream patch to fix this instead? https://github.com/mariadb-corporation/libmarias3/commit/c71898f82598

Or does this not apply cleanly?

Reply:
The issue isn't the submodule (`libmarias3`). It is how packaged `mroonga` build interacts with top-level https://github.com/MariaDB/server `VERSION` file.

`mroonga` CMake build adds `MYSQL_SOURCE_DIR == #{buildpath}` as part of includes.